NFIQ2/NFIQ2Algorithm/src/nfiq2/nfiq2_algorithm_impl.cpp

```diff
@@ -66,7 +66,7 @@ NFIQ2::Algorithm::Impl::getQualityPrediction(
 	const double utility {};
 	double deviation {};
 	double quality {};
-	m_RandomForestML.evaluate(features, utility, quality, deviation);
+	m_RandomForestML.evaluate(features, quality);
```

Comment:
I'm assuming you still get a warning above for `utility` and `deviation` being defined but not used? This would close #229.